tests/WizardTest.php

```diff
@@ -16,7 +16,9 @@ class WizardTest extends PHPUnit\Framework\TestCase
     protected $secondTestStepClass;
     protected $wizard_reflection;
     protected $thirdTestStepClass;
+    protected $fourthTestStepClass;
```

Comment:
I don't like how this test look (all file). I think i will rewrite this test and instead of create mock steps I add some dump step classes.

Reply:
ok i made a small refactor here you can check what i change -> https://github.com/smajti1/laravel-wizard/commit/a3d2367459d807f0ca6d54152df4d9f47aab95b6